BREAKING_CHANGES.md

```diff
@@ -0,0 +1,25 @@
+## Breaking Changes in Terraform provider
+
+## Version Compatibilities
+
+Our releases follow the Semantic Versioning (SemVer) convention to ensure predictability and stability. Each version is represented as MAJOR.MINOR.PATCH, where:
+
+- MAJOR version updates may contain breaking changes.
+- MINOR version updates introduce backward-compatible functionality and can contain breaking changes 
+- PATCH version updates encompass backward-compatible bug fixes.
+
+## Release Support Period
+
+Every release of the Atlas Terraform Provider is officially supported for approximately one year from its initial release date.
```

Comment:
I don't think this is true - we support the latest release of the provider. For issues, we encourage upgrading, as we don't release updates for older versions.

In line with how we support the AtlasCLI & other tools where we don't use a magor release number